examples/hellocluster/launch_hellocluster.py

```diff
@@ -1,15 +1,17 @@
 import logging
-import os
 
-from slurmpilot.config import default_cluster_and_partition
+from pathlib import Path
+
+from slurmpilot.config import default_cluster_and_partition, load_config
 from slurmpilot.slurm_wrapper import SlurmWrapper, JobCreationInfo
 from slurmpilot.util import unify
 
 if __name__ == '__main__':
     logging.basicConfig(level=logging.INFO)
+    config = load_config()
```

Comment:
Thanks!
We dont need to specify the config right? (lets merge the PR this way, I will try to run without this line and remove it if it is not needed)